src/sb_gazebo/scripts/teleop_twist_keyboard.py

```diff
@@ -0,0 +1,132 @@
+#!/usr/bin/env python
```

Comment:
Generally executable scripts don't have the `.py` extensions. 
[Ref](http://docs.ros.org/jade/api/catkin/html/howto/format2/installing_python.html

Though there's this other guide [list](http://wiki.ros.org/PyStyleGuide). Should probably figure out which one to follow going forward.